.github/workflows/master_build_push.yml

```diff
@@ -11,11 +11,9 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Setup .NET Core
+    
+    - name: Setup .NET
       uses: actions/setup-dotnet@v1
-      with:
-        dotnet-version: 3.1.101
```

Comment:
Why not pin the version to `5.0.400`? I doubt it will take the value from `global.json`

Reply:
It does take it, and that's the beauty of it. It pins for local dev and CI environments.
![image](https://user-images.githubusercontent.com/227765/133093478-84f419ca-075c-4834-8c02-0edebdbae376.png)

Reply:
[True](https://github.com/actions/setup-dotnet/blob/main/src/setup-dotnet.ts#L12)!